### ISSUE TITLE
Gremlins

### ISSUE THREAD

**Issue by Toffik5** [TeamChocoQuest/ChocolateQuestRepoured#251]
**Common sense Info**
To check a checkbox: Insert a lower case X between the square brackets, you need to remove the space though.
 - I play...
   - [x ] With a large modpack
   - [ ] Only with CQR and it's dependencies
 - The issue occurs in...
   - [x ] Singleplayer
   - [ ] Multiplayer
 - [x] I have searched for this or a similar issue before reporting and it was either (1) not previously reported, or (2) previously fixed  and I'm having the same problem.
 - [ ] I am using the latest version of the mod (all versions can be found on github under releases)
 - [ ] I read through the FAQ and i could not find something helpful ([FAQ](https://wiki.cq-repoured.net/index.php?title=FAQ))
 - [ ] I reproduced the bug without any other mod's except forge, cqr and it's dependencies
 - [ ] The game crashes because of this bug
 
 
 **Versions**
Chocolate Quest Repoured ('latest' IS NOT a valid version! Please use the version string that shows up in the mod overview of forge):
Forge: 1.12.2 14.23.5.2854
Minecraft: 1.12.2

**Describe the bug**
So basically i don't have sounds on gremlins. I saw that 1.7 better duengons mod has these sound and they work good. I dont know if they got deleted or its just bug.

**To Reproduce**
2. Use item '....'
3. Place block '....'
.
.
.

**Expected behavior**

**Screenshots**

**Other**
Please note: log files and crash reports should be uploaded as a FILE and not as a copy paste of the text of it here. Upload the file to a site like pastebin or similar and link it here!!
 - Link to Crash report file:
 - Link to Log file:
 - Other mods (that are necessary to produce the bug):
 - Other mods (that are also installed):
 - CQR config folder (link, only if you changed the configs):
 - Other mod's configs (link):

**Comment by DerToaster98** [comment]
CQR isn’t the same mod as BD/CQC

Gremlins don’t have unique sounds yet, they‘ll get some in the future
